backend/api/offenderSearchApi.js

```diff
@@ -0,0 +1,40 @@
+const contextProperties = require('../contextProperties')
+const { mapToQueryString } = require('../utils')
+
+const offenderSearchApiFactory = client => {
+  const processResponse = context => response => {
+    contextProperties.setPaginationFromPageRequest(context, response.body)
+    return response.body.content
+  }
+
+  const changeFieldNames = () => data =>
+    data.map(({ uiId, prisonerNumber, bookingId, firstName, lastName, dateOfBirth, prisonId, prisonName, status }) => ({
+      uiId,
+      offenderNo: prisonerNumber,
+      firstName,
+      lastName,
+      dateOfBirth,
+      latestBookingId: Number(bookingId),
+      latestLocationId: prisonId,
+      latestLocation: prisonName,
+      currentlyInPrison: status && status.startsWith('ACTIVE') ? 'Y' : 'N',
+      currentWorkingFirstName: firstName,
```

Comment:
in global search we used to get a row per alias, now we will just get one row.  For now, just output the first name and last name again, need to think about removing the columns...